### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: app.mjs [hunk 1]
```diff
@@ -534,9 +534,9 @@ const cmdLocation = async (_systemData, _userData, relay, ev) => {
   const location = ev.content.match(REGEX_LOCATION) ? ev.content.match(REGEX_LOCATION)[2] : ev.content.match(REGEX_LOCATION_ALT) ? ev.content.match(REGEX_LOCATION_ALT)[1] : "";
   let message = "わかりませんでした…";
   if (!!location) {
-    const geoDatas = await getLocation(location);
-    if (!!geoDatas.length) {
-      const geoData = geoDatas[0];
+    const getDataItems = await getLocation(location);
```
Comment (ikuradon) [inline app.mjs:537]: I understand that "Datas" should be "DataItems", but why did you change geo to get?

**Review by ikuradon — CHANGES_REQUESTED** [review]

File: app.mjs (hunk 1 above)
Comment (mattn) [reply app.mjs:537]: Wow, I'll fix soon

**Comment by ikuradon** [comment]
🙏
